### DISCUSSION TITLE
Link to Wiki > Legacy Installation > Github Release was altered to an outside domain. Intentional?

### DISCUSSION THREAD

**Discussion by Fanolian** [martinet101/ElevenClock#972] [General]
Link in question:
https://github.com/martinet101/ElevenClock/wiki/Home/ece569c758d4edfd6a45322d7ab6cd9544a5ed92#legacy-installation
The edit was made in Jun19.
Comparison with the previous edit, made by martinet101 on Jun15: https://github.com/martinet101/ElevenClock/wiki/Home/_compare/6842a5f00ce74d8e3a988825ca223439a1d2df31...ece569c758d4edfd6a45322d7ab6cd9544a5ed92

I just discovered this software and was reading the wiki. I find it strange that the installation link pointed to a 404 page from an outside domain. I did a simple search but couldn't find any discussions or issues about the change. Is it intentional?

The author of the change (Iightspark) seems to have deleted their Github account: https://github.com/Iightspark. I couldn't find any issues or discussions in this repo started by that author neither.

I jumped the gun and corrected the link (https://github.com/martinet101/ElevenClock/wiki/Home/_history) . If that change was intentional, please let me know and revert my change. Thank you.

**Comment by marticliment** [comment]
Well, I did not know that, thank you for realising and modifying!

This was not an expected change, and while the url seems like a data harvesting url, i don't know who did that (and neither have I received any information about that.)

Gonna restrict the wiki edits to contributors only for more safety.

**Comment by Fanolian** [comment]
I pasted an extra `(` in https://github.com/martinet101/ElevenClock/wiki#running-and-building-the-source-code. Please fix for me😅. Thanks.

**Reply by marticliment** [reply]
Done!
